speedcenterBuild

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+set -e
+cd $(dirname $0)
```

Comment:
```suggestion
cd "$(dirname "$0")"
```
Quoting is fun! This currently fails if the file is in a directory with a space in the name.